File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/NumarParticipantiSedinteComisieGlReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class NumarParticipantiSedinteComisieGlReportFilterModel {

	private Date dataSedintaDeLa;
	private Date dataSedintaPanaLa;
	private Long comisieId;
	private Long bancaId;
	private String functie;
	private String departament;
	private String calitateMembru;
	private Long responsabilId;

	public Date getDataSedintaDeLa() {
		return dataSedintaDeLa;
	}

	public void setDataSedintaDeLa(Date dataSedintaDeLa) {
		this.dataSedintaDeLa = dataSedintaDeLa;
	}

	public Date getDataSedintaPanaLa() {
		return dataSedintaPanaLa;
	}

	public void setDataSedintaPanaLa(Date dataSedintaPanaLa) {
		this.dataSedintaPanaLa = dataSedintaPanaLa;
	}

	public Long getComisieId() {
		return comisieId;
	}

	public void setComisieId(Long comisieId) {
		this.comisieId = comisieId;
	}

	public Long getBancaId() {
		return bancaId;
	}

	public void setBancaId(Long bancaId) {
		this.bancaId = bancaId;
	}

	public String getFunctie() {
		return functie;
	}

	public void setFunctie(String functie) {
		this.functie = functie;
	}

	public String getDepartament() {
		return departament;
	}

	public void setDepartament(String departament) {
		this.departament = departament;
	}

	public String getCalitateMembru() {
		return calitateMembru;
	}

	public void setCalitateMembru(String calitateMembru) {
		this.calitateMembru = calitateMembru;
	}

	public Long getResponsabilId() {
		return responsabilId;
	}

	public void setResponsabilId(Long responsabilId) {
		this.responsabilId = responsabilId;
	}

}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/numar-participanti-sedinte-comisie-gl-report-filter.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class NumarParticipantiSedinteComisieGlReportFilterModel {
    
	@JsonProperty("dataSedintaDeLa", JsonDateConverter)
    public dataSedintaDeLa: Date = null;
    
	@JsonProperty("dataSedintaPanaLa", JsonDateConverter)
    public dataSedintaPanaLa: Date = null;
    
	@JsonProperty("comisieId", Number)
	public comisieId: number = null;
    
	@JsonProperty("bancaId", Number)
	public bancaId: number = null;
    
	@JsonProperty("functie", String)
	public functie: string = null;
    
	@JsonProperty("departament", String)
	public departament: string = null;
    
	@JsonProperty("calitateMembru", String)
	public calitateMembru: string = null;
    
	@JsonProperty("responsabilId", Number)
	public responsabilId: number = null;
}